timescaledb/tutorials/grafana/visualizations/candlestick.md

```diff
@@ -0,0 +1,167 @@
+
+Candlestick charts use 4 different aggregations over a specific time interval: open, high, low, and close values. 
+
+* **Open**: The starting value of a given period.
+
+* **High**: The highest value of a given period.
+
+* **Low**: The lowest value of a given period.
+
+* **Close**: The final value of a given period.
+
+These values are used by the candlestick panel to visualize your data. 
+
+A candlestick chart shows a clear relationship between the opening and closing prices in a given time frame: 5-minute, 10-minute, 1-hour, etc.  Within a given time period, candlestick charts enable people to quickly identify patterns in the changing price of an asset. 
+For example, candlestick charts can identify whether an asset is entering the bullish or bearish territory, if the market activity for an asset is topping or bottoming out, or other characteristics that can help traders identify trends.  
+
+## What you will learn
+This tutorial shows you how to:
+
+*   [Create candlestick aggregates with raw data](#Create-a-candlestick-with-raw-data)
+*   [Include volume when querying with raw data](#Include-volume-when-querying-from-raw-data)
+
+## Prerequisites
+Before you begin, make sure you have:
+
+* Installed [Grafana][install-grafana] version&nbsp;8.5 or higher
+* Installed [TimescaleDB][install-timescale]
+* Imported the stock trade data from the [Getting Started Tutorial][gsg-data]
+
+If you are new to Grafana, see the
+[Grafana tutorials][grafana-tutorials]
+to get familiar with creating your first dashboard and visualizations before you
+start.
+
+The examples in this section use these variables and Grafana functions:
+* `$symbol`: a variable used to filter results by stock symbols.
+* `$__timeFrom()::timestamptz` & `$__timeTo()::timestamptz`:
+  Grafana variables. You change the values of these variables by
+  using the dashboard's date chooser when viewing your graph.
+* `$bucket_interval`: the interval size to pass to the `time_bucket`
+  function when aggregating data.
+
+To learn more about adding variables to Grafana, be sure to check out this tutorial
+
+## Create a candlestick with raw data
+Once you have the data and completed the prerequisites, we can create a Candlestick visualization using the raw stock tick data in the table `stocks_real_time`.
+
+<procedure>
+
+### Create a candlestick with raw data
+
+1.  In the query editor, use this SQL to query a Candlestick dataset using specified bucket interval:
+     ```sql
+    SELECT
+    time_bucket($bucket_interval, time) AS time,
+    symbol,
+    FIRST(price, time) AS "open",
+    MAX(price) AS high,
+    MIN(price) AS low,
+    LAST(price, time) AS "close",
+    FROM stocks_real_time
+    WHERE symbol =  $symbol
+    AND time > $__timeFrom()::timestamptz and time < $__timeTo()::timestamptz
+    GROUP BY symbol;
+    ```
+1.  Click outside of the query editor/the refresh icon to 
+    update the Grafana chart.
+
```

Comment:
This may be a bit too trivial to include, but you are more familiar with Grafana than I am. Is it easy for people to get confused because they don't know that they need to refresh the chart?

Reply:
Yes, they won't know they need to refresh, cause you will think it automatically refreshes once you have an updated query